Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -268,6 +274,9 @@ function WebMapServiceImageryProvider(options) {
   pickFeatureResource.setQueryParameters(pickFeatureParams, true);
 
   this._resource = resource;
+
+  // TODO: change this pickFeatureResource to something like options.getFeaturesUrl if this option exists, else use the getCapabilities default URL
```

Comment:
Is this TODO comment still relevant?

Reply:
Thank you. I will remove the comment in the next update.